tests/po/formPage.ts

```diff
@@ -0,0 +1,44 @@
+import { Page, Locator} from '@playwright/test'
+
+export class FormPage {
+    readonly page : Page;
```

Comment:
readonly createPostHeader = this.page.getByTestId("page-title");
readonly createPostTOSNudge = this.page.getByTestId("nudge-forum");